<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandsEditor</class>
 <widget class="QFrame" name="CommandsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>422</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbCommands">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Commands</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QToolButton" name="tbCommandAdd">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/tools/icons/tools/add.png</normaloff>:/tools/icons/tools/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tbCommandRemove">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/tools/icons/tools/remove.png</normaloff>:/tools/icons/tools/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="tbCommandUp">
        <property name="text">
         <string>Up</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/tools/icons/tools/up.png</normaloff>:/tools/icons/tools/up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="tbCommandDown">
        <property name="text">
         <string>Down</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/tools/icons/tools/down.png</normaloff>:/tools/icons/tools/down.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QListWidget" name="lwCommands"/>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="cbCommandTypes">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbEditor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Editor</string>
     </property>
     <layout class="QHBoxLayout" name="_8">
      <item>
       <layout class="QGridLayout" name="_9">
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lCommandText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Text :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="leCommandText"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="lCommandCommand">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Command :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="leCommandCommand"/>
        </item>
        <item row="3" column="1">
         <widget class="QToolButton" name="tbCommandCommand">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/tools/icons/tools/file.png</normaloff>:/tools/icons/tools/file.png</iconset>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="lCommandArguments">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Arguments :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLineEdit" name="leCommandArguments"/>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLabel" name="lCommandWorkingDirectory">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Working Directory :</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLineEdit" name="leCommandWorkingDirectory"/>
        </item>
        <item row="7" column="1">
         <widget class="QToolButton" name="tbCommandWorkingDirectory">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/tools/icons/tools/icon.png</normaloff>:/tools/icons/tools/icon.png</iconset>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QCheckBox" name="cbCommandSkipOnError">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Skip On Error</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="lCommand">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="_10">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="lCommandParsers">
          <property name="text">
           <string>Parsers :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lwCommandParsers">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbCommandTryAll">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Try All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
